packages/web/src/js/Tooltip.ts

```diff
@@ -322,9 +353,79 @@ class Tooltip extends BaseComponent {
       });
     }
   }
+
+  autoCloseHandler = (event: Event) => {
+    const id = this.tip.getAttribute('id') as string;
+    const trigger = document.querySelector(`[data-spirit-target="#${id}"]`) as HTMLElement;
+
+    const shouldClose = trigger && clickOutsideElement(trigger, event);
+    if (event.target && shouldClose) {
+      this.activeTrigger[TRIGGER_CLICK] = false;
```

Comment:
These 2 lines are also done in `this.hide` so maybe we do not need them here.

Reply:
hm, i tried to remove it and it does not work without it